src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -0,0 +1,315 @@
+from _gettsim.piecewise_functions import piecewise_polynomial
+from _gettsim.shared import dates_active
+
+
+def erwerbsm_rente_m(
+    erwerbsm_rente_zugangsfaktor: float,
+    entgeltp_erwerbsm_rente: float,
+    rentenwert: float,
+    rentenartfaktor_erwerbsm_rente: float,
+    ges_rente_vorauss_erwerbsm: bool,
+) -> float:
+    """Calculating the Erwerbsminderungsrente pension claim
+
+    Legal reference: SGB VI § 64: Rentenformel für Monatsbetrag der Rente
+
+
+    Parameters
+    ----------
+    erwerbsm_rente_zugangsfaktor
+        See :func:`erwerbsm_rente_zugangsfaktor`.
+    entgeltp_erwerbsm_rente
+        See :func:`entgeltp_erwerbsm_rente`.
+    rentenwert
+        See :func:`rentenwert`.
+    rentenartfaktor_erwerbsm_rente
+        See :func:`rentenartfaktor_erwerbsm_rente`.
+    ges_rente_vorauss_erwerbsm
+        See :func:`ges_rente_vorauss_erwerbsm`.
+    Returns
+    -------
+    Erwerbsminderungsrente pension claim
+
+    """
+
+    if ges_rente_vorauss_erwerbsm:
+        out = (
+            entgeltp_erwerbsm_rente
+            * erwerbsm_rente_zugangsfaktor
+            * rentenwert
+            * rentenartfaktor_erwerbsm_rente
+        )
+    else:
+        out = 0.0
+    return out
+
+
+def ges_rente_vorauss_erwerbsm(
+    erwerbsgemindert: bool,
+    m_pflichtbeitrag: float,
+    ges_rente_wartezeit_5: bool,
+) -> bool:
+    """
+    Determine eligibility Erwerbsminderungsrente
+    (pension for reduced earning capacity)
+
+    Requirements are:
```

Comment:
This section can go. It is mostly documenting other functions, which are documented elsewhere. So better to look them up directly.

The only relevant thing would be the exception in `m_pflichtbeitrag`. But I'd actually be in favor of a separate input variable for that one. If someone is comfortable to take the shortcut that you did here, it is trivial to do so outside of GETTSIM.